From 6e2e55fbc045c9eaf8e601a4c28194d4a1988a48 Mon Sep 17 00:00:00 2001
From: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Date: Mon, 18 Mar 2019 11:35:55 -0700
Subject: [PATCH 0575/1852] v4l: xilinx: scd: Fix video format configuration
 for YUV420

commit 593f66a330ef1c2e3748a1bc3fce3f72e37e1b1b from
https://github.com/Xilinx/linux-xlnx.git

In the current implementation streaming based driver, the video format
value for YUV420 is programmed wrongly. This patch programs it correctly
according to PG322.

Signed-off-by: Satish Kumar Nagireddy <satish.nagireddy.nagireddy@xilinx.com>
Reviewed-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-scenechange-channel.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-scenechange-channel.c b/drivers/media/platform/xilinx/xilinx-scenechange-channel.c
index aa1b98d38a90..ffa63b46ff24 100644
--- a/drivers/media/platform/xilinx/xilinx-scenechange-channel.c
+++ b/drivers/media/platform/xilinx/xilinx-scenechange-channel.c
@@ -51,7 +51,7 @@
 #define XSCD_COLOR_FMT_RGB		0
 #define XSCD_COLOR_FMT_YUV_444		1
 #define XSCD_COLOR_FMT_YUV_422		2
-#define XSCD_COLOR_FMT_YUV_420		4
+#define XSCD_COLOR_FMT_YUV_420		3
 
 #define XSCD_V_SUBSAMPLING		16
 #define XSCD_BYTE_ALIGN			16
-- 
2.31.1

